.github/workflows/release.yml

```diff
@@ -58,7 +58,7 @@ jobs:
 
     needs: build-linux
 
-    runs-on: [self-hosted, Windows, X64]
+    runs-on: [self-hosted, Windows, X64, opam-2-3]
```

Comment:
AFAIK adding an action runner with opam 2.3 installed will break older builds, even if the `opam-2-3` tag is set (because the tags are matched by checking whether they are a subset).

Should we even bother with the tag, then?

Reply:
This tag allows forward compatibility with future versions of opam: if I need to create a new agent for a new version of opam that is incompatible with opam 2.3 and requires overhaul of the GitHub workflow, CI scripts, etc., then old and new CI should not use each other's agents.
In practice, I added this tag to ensure that the new Everest CI for opam 2.3 wouldn't choose then-existing [fdopen](https://fdopen.github.io/opam-repository-mingw/)-based agents.